evap/staff/templates/staff_course_person_management.html

```diff
@@ -72,17 +72,39 @@ <h6 class="card-subtitle mb-2 text-muted">{% trans 'From Excel file' %}</h6>
             </form>
         </div>
         <div class="col">
-            <form id="contributor-copy-form" method="POST" enctype="multipart/form-data" class="form-vertical">
+            <form id="contributor-copy-form" method="POST" enctype="multipart/form-data" class="form-vertical h-100">
                 {% csrf_token %}
-                <div class="card">
+                <div class="card h-100">
                     <div class="card-body import-card">
                         <h4 class="card-title">{% trans 'Copy Contributors' %}</h4>
                     <h6 class="card-subtitle mb-2 text-muted">{% trans 'From other course' %}</h6>
                         <p class="card-text">{% trans 'Copy contributors from another course.' %}</p>
                         {% include 'bootstrap_form.html' with form=contributor_copy_form wide=True %}
                     </div>
                     <div class="card-footer text-center">
-                        <button name="operation" value="copy-contributors" type="submit" class="btn btn-sm btn-primary form-submit-btn">{% trans 'Copy Contributors' %}</button>
+                        <button name="operation" value="copy-contributors" type="submit" class="btn btn-sm btn-primary form-submit-btn">{% trans 'Copy contributors' %}</button>
+                    </div>
+                </div>
+            </form>
+        </div>
+    </div>
+
+    <div class="row">
+        <div class="col">
+            <form id="login-key-export-form" method="POST" class="form-vertical" action="{% url 'staff:course_login_key_export' semester.id course.id %}">
+                {% csrf_token %}
+                <div class="card">
+                    <div class="card-body import-card">
+                        <h4 class="card-title">{% trans 'Export Login Keys' %}</h4>
+                        <h6 class="card-subtitle mb-2 text-muted">{% trans 'To Excel File' %}</h6>
```

Comment:
it's actually not an excel file, but a csv file. can you please change that here and below?